### PR TITLE
chore(react-query): update devDeps

### PR DESCRIPTION
**Pull request by manudeli**

### CODE REVIEW

File: packages/react-query/package.json [hunk 1]
```diff
@@ -65,12 +65,12 @@
   },
   "devDependencies": {
     "@tanstack/query-persist-client-core": "workspace:*",
-    "@testing-library/react": "^16.0.1",
+    "@testing-library/react": "^16.1.0",
     "@testing-library/react-render-stream": "^2.0.0",
-    "@types/react": "npm:types-react@rc",
-    "@types/react-dom": "npm:types-react-dom@rc",
```
Comment (manudeli) [inline packages/react-query/package.json:71]: This can be stable version now!

**Review by TkDodo — APPROVED** [review]
Thx. I forgot those

**Comment by TkDodo** [comment]
FYI:

```
1 issue found in ./:

 ⨯ error Dependency @testing-library/react has multiple versions defined in the workspace. multiple-dependency-versions
  ./packages
      react-query-devtools      ^16.0.1   ↓ lowest
      react-query-persist-client   ^16.0.1   ↓ lowest
      react-query               ^16.1.0   ↑ highest

1 issue found (1 ⨯, 0 ⚠️, 0 ✓) across 82 packages in 2.30253ms.
```

**Comment by TkDodo** [comment]
@manudeli there are conflicts now

**Comment by manudeli** [comment]
> @manudeli there are conflicts now

I'll resolve it now!

**Comment by manudeli** [comment]
@TkDodo I resolved it now!

**Pull request merged**